test/testrunner.py

```diff
@@ -145,13 +149,15 @@ def main():
     LOGGER.info(f"Images tested: {dlc_images}")
     all_image_list = dlc_images.split(" ")
     standard_images_list = [image_uri for image_uri in all_image_list if "example" not in image_uri]
-    new_eks_cluster_name = None
+    eks_cluster_name = None
     benchmark_mode = "benchmark" in test_type
     specific_test_type = re.sub("benchmark-", "", test_type) if benchmark_mode else test_type
     test_path = os.path.join("benchmark", specific_test_type) if benchmark_mode else specific_test_type
 
     if specific_test_type in ("sanity", "ecs", "ec2", "eks"):
         report = os.path.join(os.getcwd(), "test", f"{test_type}.xml")
+        report_train = os.path.join(os.getcwd(), "test", f"{test_type}_train.xml")
```

Comment:
I see that you're trying to make this extensible to other test types, but it could get confusing as to why this is here. Can you add a clarifying comment, and note that only eks tests leverage this currently

Reply:
Makes sense. I can add this.